package.json

```diff
@@ -2,6 +2,9 @@
   "name": "@event-driven-io/core",
   "version": "0.1.7",
   "description": "Emmett - Event Sourcing development made simple",
+  "engines": {
+    "node": ">=21.6.2"
```

Comment:
Hey, @oskardudycz - this is something that needs you attention. In order to use Node's test runner along with its latest features, we need to use the "Current" version of [Node](https://nodejs.org/en). Let me know if that's okay

Reply:
@thiagomini my main concern is not to lost or be able to catch other 21.x specifics for packages so as not to accidentally make it incompatible with the LTS version. Afaik some runtimes like AWS Lambda doesn't support 21.x yet and I’d like to have it compatible.

I know that for now it’ll be used only in tests, so in theory that’s fine. 

Thoughts?

Reply:
I understand your concern, @oskardudycz - There's a way to use LTS here, but we'd have to change the `test` script in the root package.json. I noticed you changed it to not run the `test` script of each workspace, but rather to execute the tests directly - this doesn't work with LTS version of node because the test runner doesn't recognize glob patterns yet (see https://github.com/nodejs/node/issues/50658).

An alternative would be to roll back the root package.json so it calls each workspace "test" script

Reply:
I did that to make possible running the Jest plugin for tests and enable debugging. I don’t mind changing that if it makes it work. Did you happen to check tests debugging capabilities?

Reply:
I see, Oskar! Yea, it's possible to debug a test, we just need to attach a debugger - or if you are using vscode, open a "Javascript debugger terminal" and execute the test command, it will automatically attach a debugger and stop on breakpoints